src/Integration.Vsix/Suppression/SuppressionManager.cs

```diff
@@ -96,43 +96,14 @@ private void OnSolutionBindingChanged(object sender, ActiveSolutionBindingEventA
         private bool ShouldIssueBeReported(SyntaxTree syntaxTree, Diagnostic diagnostic)
         {
             // This method is called for every analyzer issue that is raised so it should be fast.
-            if (!diagnostic.Location.IsInSource &&
-                diagnostic.Location != Location.None)
-            {
-                return true;
-            }
 
             if (activeSolutionBoundTracker == null ||
```

Comment:
I left this check in the _SuppressionManager_ so that the suppression handler doesn't need to know about the solution tracker.